### DISCUSSION TITLE
Cannot share to nextcloud through WebDAV

### DISCUSSION THREAD

**Discussion by tjandras** [photoprism/photoprism#3781] [Q&A]
Hi!

I just started my instance with docker, I could index my originals, everything looks fine. But I'm having problems with sharing to NextCloud.

 I added a service for my NextCloud instance, which seems to work until I try to share some photos to it. I can browse the folders to which I would like to share, but the actual upload does not happen. The file appears in NextCloud at the correct folder, but the size is zero. I only see error in PhotoPrism logs, not in NextCloud. The error I get is this:

`webdav: 'io: read/write on closed pipe' `

Any ideas what can be the problem, and how to solve it? 

Thanks in advance!

**Comment by lastzero** [comment]
What PhotoPrism version are you using? Do you run it behind reverse proxy like nginx?

**Reply by tjandras** [reply]
Version: 230923-ce
NextCloud is behind nginx. Photoprism is not, running it only locally.

**Reply by lastzero** [reply]
AFAIK nginx isn't compatible with Nextcloud when using WebDAV.

**Reply by lastzero** [reply]
https://github.com/nextcloud/server/issues/7995

**Reply by tjandras** [reply]
I see. Thanks for the info and link.
